wwr_ui/randomizer_window.ui

```diff
@@ -1171,6 +1171,31 @@
               </layout>
              </widget>
             </item>
+            <item>
+             <widget class="QGroupBox" name="groupBox_logic_tweaks">
+              <property name="title">
+               <string>Logic Tweaks</string>
```

Comment:
Maybe this should be called something different since it affects things in game and not just the logic? This option seems similar to "Skip Boss Rematches", but isn't really a convenience tweak